docs/reference/roles_matrix.rst

```diff
@@ -0,0 +1,47 @@
+Roles Matrix
+============
+
+The ``Sonata\UserBundle\Form\Type\RolesMatrixType`` was built to show all roles in a matrix view.
+
+![Roles matrix](../images/roles_matrix.png)
+
+Every admin has defined default roles like:
+
+    - EDIT
+    - LIST
+    - CREATE
+    - VIEW
+    - DELETE
+    - EXPORT
+    - ALL
+
+The roles matrix consists of two parts:
+
+1. shows the matrix with each admin and their permissions.
+2. shows the custom roles which are configured in ``security.yml`` and list them as checkboxes (+ showing their inherited roles).
+
+.. note::
+
+    If the current logged in user is not granted to an admin the checkbox is disabled.
```

Comment:
If the current user is not admin? I am not sure I understand this sentence.

Reply:
I don't understand it either, please rephrase.